src/Entity/EntityMetaRelationType.php

```diff
@@ -0,0 +1,63 @@
+<?php
+
+declare(strict_types = 1);
+
+namespace Drupal\emr\Entity;
+
+use Drupal\Core\Config\Entity\ConfigEntityBundleBase;
+
+/**
+ * Defines the Entity Meta Relation type configuration entity.
+ *
+ * @ConfigEntityType(
+ *   id = "emr_type",
```

Comment:
OK, I understand where my confusion was from. This entity type is called `emr_type` whereas the one from the meta entitites is called `entity_meta_type`. I think this is kinda weird. Let's just have the module name and namespace be the short one `emr`. And for the config entity type we call them what they are: `entity_meta_relation_type` (`entity_meta_relation`) and `entity_meta_type` (`entity_meta`)